Rasa_Bot/actions/actions.py

```diff
@@ -422,6 +423,57 @@ def _validate_now_or_later_response(value):
         if value.lower() in ['later', 'later.', 'niet nu']:
             return False
         return None
+    
+
+class ActionGetReschedulingOptionsList(Action):
+    """Get the possible rescheduling options."""
+    
+    def name(self):
+        return "action_get_rescheduling_options_list"
+    
+    async def run(self, dispatcher, tracker, domain):
+        
+        # define morning, afternoon, evening
+        morning = [6, 12]
+        afternoon = [12, 18]
+        evening = [18, 24]
```

Comment:
```suggestion
        # define morning, afternoon, evening
        MORNING = (6, 12)
        AFTERNOON = (12, 18)
        EVENING = (18, 24)
```
Would be nice to mark these as constants (by capitalizing), and use tuples instead of lists, because they should be unmutable.